src/wallet/wallet_rpc_server.cpp

```diff
@@ -4517,7 +4517,6 @@ class t_daemon
 
       const auto arg_wallet_file = wallet_args::arg_wallet_file();
       const auto arg_from_json = wallet_args::arg_generate_from_json();
-      const auto arg_rpc_client_secret_key = wallet_args::arg_rpc_client_secret_key();
```

Comment:
I am a bit confused: If I checked correctly you let a lot of RPC payment related stuff stand in the wallet RPC server, but out all possible things to delete you only delete things related to this key.

If that's really the case, what's the idea here?